app/assets/stylesheets/custom.css.scss

```diff
@@ -179,6 +187,27 @@ img {
 
 // Organization index
 
+.org-search {
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+}
+
+#q {
+  // background-color: scale-color($share-dark-blue, $lightness: 25%);
+  // color: $ghost;
+  height: 45px;
+}
+
+.exit {
+  color: $share-yellow;
```

Comment:
Properties should be ordered background-color, color, margin-left, vertical-align